docs/management/connectors/action-types/cases-webhook.asciidoc

```diff
@@ -0,0 +1,179 @@
+     updateIncidentUrl: 'https://test.host/rest/api/2/issue/{{{external.system.id}}}'
+     updateIncidentMethod: 'put'
+     updateIncidentJson: '{"fields":{"summary":{{{case.title}}},"description":{{{case.description}}},"labels":{{{case.tags}}}'
+     createCommentMethod: 'post',
+     createCommentUrl: 'https://test.host/rest/api/2/issue/{{{external.system.id}}}/comment',
+     createCommentJson: '{"body": {{{case.comment}}}}',
+   secrets:
+     user: testuser
+     password: passwordkeystorevalue
+--
+
+Config defines information for the connector type.
+
+`hasAuth`:: A boolean that corresponds to *Requires authentication*. If `true`, this connector will require values for `user` and `password` inside the secrets configuration. Defaults to `true`.
+`headers`:: A record<string, string> that corresponds to *Headers*.
+`createIncidentUrl`:: A URL string that corresponds to *Create Case URL*.
+`createIncidentMethod`:: A string that corresponds to *Create Case Method*.
+`createIncidentJson`:: A stringified JSON with mustache variables that corresponds to *Create Case JSON*.
+`createIncidentResponseKey`:: A string from the response body of the create case method that corresponds to the *External Service Id*.
+`getIncidentUrl`:: A URL string with External Service Id mustache variable that corresponds to *Get Case URL*.
+`getIncidentResponseExternalTitleKey`:: A string from the response body of the get case method that corresponds to the *External Service Title*.
+`getIncidentResponseCreatedDateKey`:: A string from the response body of the get case method that corresponds to the *External Service Case Created Date*.
+`getIncidentResponseUpdatedDateKey`:: A string from the response body of the get case method that corresponds to the *External Service Case Updated Date*.
+`viewIncidentUrl`:: A URL string with either the External Service Id or External Service Title mustache variable that corresponds to *View Case URL*.
+`updateIncidentUrl`:: A URL string that corresponds to *Update Case URL*.
+`updateIncidentMethod`:: A string that corresponds to *Update Case Method*.
+`updateIncidentJson`:: A stringified JSON with mustache variables that corresponds to *Update Case JSON*.
+`createCommentUrl`:: A URL string that corresponds to *Create Comment URL*.
+`createCommentMethod`:: A string that corresponds to *Create Comment Method*.
+`createCommentJson`:: A stringified JSON with mustache variables that corresponds to *Create Comment JSON*.
+
+
+Secrets defines sensitive information for the connector type.
+
+`user`:: A string that corresponds to *User*. Required if `hasAuth` is set to `true`.
+`password`:: A string that corresponds to *Password*. Required if `hasAuth` is set to `true`.
+
+[float]
+[[define-webhook-ui]]
+==== Define connector in Stack Management
+
+Define Webhook - Case Management connector properties.
+
+[role="screenshot"]
+image::management/connectors/images/cases-webhook-connector.gif[Webhook - Case Management connector]
```

Comment:
It will be more clear if this gifs will correspond to the example steps.

Reply:
like you mean the same values? i was trying to create a new gif but my software keeps freezing...

Reply:
I can help you with recording the example gifs and URLs/bodies , I have it is working somewhere on my remote cluster.